versioned_sidebars/version-3.0.0-beta-sidebars.json

```diff
@@ -68,6 +68,11 @@
       "label": "Templates",
       "items": ["templates/koii-template"]
     },
+    {
+      "type": "category",
+      "label": "Tutorial",
```

Comment:
either name this Quick Start or Tutorial. let's have only one category for this

Reply:
@rekpero  i want all tutorials to come under this tutorial category...

except if i don't understand what you mean here

Reply:
@seunbayo We have a quick start which is similar to the tutorial category so I wanted to make it consistent. Either we use quick start or tutorial not both